src/scss/App.scss

```diff
@@ -0,0 +1,184 @@
+$background-color-main: #F4F4F4;
+$background-color-purple: #D0073E;
+$background-color-grey: #EDEDED;
+$text-dark-grey: #919191;
+$border-shadow: #D4D3D3;
+$border-dark-grey: #919191;
+$grey-three: #919191;
+$white: white;
```

Comment:
Let's try not to use color names for variables, always use 6 digit hex or rbga formats:

`$white: #FFFFFF`